packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx

```diff
@@ -149,73 +154,64 @@ export const DeckLayout = ({
-            <Deck.Root
-              classNames={[
-                !flatDeck && 'surface-deck',
-                layoutMode === 'deck' && [
-                  'absolute inset-0',
-                  'transition-[padding] duration-200 ease-in-out',
-                  slots?.wallpaper?.classNames,
-                ],
-              ]}
-              solo={layoutMode === 'solo'}
-              style={{ ...overscrollAmount }}
-            >
-              {layoutMode === 'solo' &&
-                layoutParts.solo?.map((layoutEntry) => {
-                  return (
-                    <Plank
-                      key={layoutEntry.id}
-                      entry={layoutEntry}
-                      layoutParts={layoutParts}
-                      part='solo'
-                      flatDeck={flatDeck}
-                    />
-                  );
-                })}
-              {layoutMode === 'deck' &&
-                layoutParts.main?.map((layoutEntry) => {
-                  return (
-                    <Plank
-                      key={layoutEntry.id}
-                      entry={layoutEntry}
-                      layoutParts={layoutParts}
-                      part='main'
-                      resizeable
-                      flatDeck={flatDeck}
-                      searchEnabled={searchEnabled}
-                    />
-                  );
-                })}
-            </Deck.Root>
+            <div role='none' className={layoutMode === 'solo' ? 'contents' : 'relative'}>
+              <Deck.Root
+                ref={deckRef}
+                solo={layoutMode === 'solo'}
+                style={padding}
+                classNames={[
+                  !flatDeck && 'surface-deck',
+                  layoutMode === 'deck' && [
+                    'absolute inset-0',
+                    'transition-[padding] duration-200 ease-in-out',
+                    slots?.wallpaper?.classNames,
+                  ],
+                ]}
+              >
+                {parts.map((layoutEntry) => (
+                  <Plank
+                    key={layoutEntry.id}
+                    entry={layoutEntry}
+                    layoutParts={layoutParts}
+                    part={layoutMode === 'solo' && layoutEntry.id === activeId ? 'solo' : 'main'}
+                    flatDeck={flatDeck}
+                    searchEnabled={!!searchPlugin}
+                    resizeable={layoutMode === 'deck'}
+                    classNames={layoutMode === 'deck' || layoutEntry.id === activeId ? '' : 'hidden'}
+                  />
+                ))}
+              </Deck.Root>
+            </div>
           </Main.Content>
         )}
 
+        {/* TODO(burdon): Remove? */}
         {/* Note: This is not Main.Content */}
         <Main.Content role='none' classNames={['fixed inset-inline-0 block-end-0 z-[2]']}>
           <Surface role='status-bar' limit={1} />
         </Main.Content>
```

Comment:
The status bar is how we invoke the status panel, the discord link, and the feedback popover. We should definitely discuss before removing.

Reply:
oh i didn't know what this is. there's some other old things here -- like the hints.
I thought this was the "old" main.